### PR TITLE
Work on login fix and reposition buttons on add project form and clear state on login

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

Fix login, realign project buttons, remove useless code

#### Description of Task to be completed?
- Work on feedback and delete redundant code

#### How should this be manually tested?

* Check it out here.

#### Screenshots
image

### CODE REVIEW

**Comment by Mubangizi** [comment]
Hey @neelxie  Please Specify What your going to work on Exactly, Which Part of the Feed back are you planning on attempting?

**Comment by neelxie** [comment]
> Hey @neelxie Please Specify What your going to work on Exactly, Which Part of the Feed back are you planning on attempting?
> In the title of your PR

@Mubangizi thank you for the feedback have, I hope it is more elaborate now.

**Comment by Mubangizi** [comment]
> * Brilliant work on clearing the state... fist_oncoming
> * However, the PR looks like its handling two very distinct things... consider separating these concerns next time.
> * clapclapclap

Yeah @neelxie  you should not stray away from your feature, as @StephenAraka  mentioned. Great work though

File: src/redux/actions/ClustersActions.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const startFetchingClusters = () => ({
 
 export const getClustersSuccess = (response) => ({
   type: GET_CLUSTERS,
-  payload: response.data.data,
+  payload: response.data.data.clusters,
```
Comment (Mubangizi) [inline src/redux/actions/ClustersActions.js:13]: And you by changing this line you caused Errors on the `Admin side`, They depend on this object and now app just crashes, Please remove it or refactor all instances of it, Thank You

Comment (neelxie) [reply src/redux/actions/ClustersActions.js:13]: @Mubangizi this has been fixed, kindly do check it out boss.

**Comment by neelxie** [comment]
> Okay @neelxie Your PR does something way `Different` from what your description says. Please Revise it.
@Mubangizi @StephenAraka this was done to prevent having "one-line" PR because as you can see, most of the changes were one lines.

**Pull request merged**